mosquitto/rootfs/etc/cont-init.d/nginx.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Configures NGINX
+# ==============================================================================
+# This template only uses environment vars, no input
+echo "{}" \
+  | tempio \
+    -template /etc/nginx/nginx.gtpl \
```

Comment:
We streamline that on all plugin + add-on to work with `/usr/share/tempio` to have all templates in one place like i.e. : https://github.com/home-assistant/addons/tree/master/homematic/rootfs/usr/share/tempio - I would like to keep that over the org as same